docs/11-circleci/05-executors.mdx

```diff
@@ -0,0 +1,455 @@
+---
+toc_max_heading_level: 4
+---
+
+# Executors
+
+The orb provides reusable execution environments for you to run your jobs on. The executors
+available are [macOS](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos),
+[macOS on Runner](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-macos-runner),
+[Windows](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-windows-2022),
+[Windows on Runner](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-windows-runner)
+and [Ubuntu on Docker](https://circleci.com/developer/orbs/orb/game-ci/unity#executors-ubuntu).
+
+Due to the installation of external dependencies, the `macos` and `windows` executors have longer
+environment preparation times. This can be mitigated using their
+[Runner](https://circleci.com/docs/runner-overview) variants. As Runner doesn't share the ephemeral
+nature of the CircleCI cloud executors, the external dependencies will already be installed after
+the first run, decreasing the environment preparation time.
```

Comment:
```suggestion
The `macos` and `windows` executors take longer to setup due to the installation of external dependencies. 
This can be mitigated using their
[Runner](https://circleci.com/docs/runner-overview) variants. As Runner doesn't share the ephemeral
nature of the CircleCI cloud executors, the external dependencies will already be installed after
the first run, decreasing the environment preparation time.
```